modules/apps/frontend-taglib/frontend-taglib-clay/TEST-frontend-js.xml

```diff
@@ -1,11 +1,50 @@
 <?xml version="1.0" encoding="UTF-8"?>
-<testsuite hostname="" id="0" errors="0" failures="0" name="Jest" package="" skipped="0" tests="4" time="0" timestamp="1545241536984">
-  <testcase classname="Jest.ManagementToolbar" name="should listen to the rowToggled event from the registered search container" time="0.049">
+<testsuite hostname="" id="0" errors="0" failures="4" name="Jest" package="frontend-taglib.frontend-taglib-clay" skipped="0" tests="4" time="0" timestamp="1545258656167">
+  <testcase classname="frontend-taglib.frontend-taglib-clay.test.management_toolbar.ManagementToolbar" name="ManagementToolbar should listen to the rowToggled event from the registered search container" time="0.046">
+    <failure message="Jest:  Expected mock function to have been called.">      56 | 				// searchContainer.fire('rowToggled');
+      57 | 
+    > 58 | 				expect(searchContainerCallbacks.rowToggled).toHaveBeenCalled();
+      59 | 			}
+      60 | 		);
+      61 | 
+      
+      at Object.toHaveBeenCalled (test/management_toolbar/ManagementToolbar.es.js:58:49)
+</failure>
```

Comment:
-Include failure child tags where any individual js test fails.
-Include message that contains the "Expected..." line or two.
-Between failure tags, include jest output stacktrace